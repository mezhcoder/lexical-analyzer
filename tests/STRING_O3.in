#65#66
#80#81
#82