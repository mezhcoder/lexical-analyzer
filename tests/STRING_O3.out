1    1    STRING    AB
2    1    STRING    PQ
3    1    CHAR    R
3    2    EOF    EOF